apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -171,22 +181,37 @@ export function ListBoxInline({ app, fieldIdentifier, stateName = '$', options =
 
   const minHeight = 49 + (search ? 40 : 0) + 49;
```

Comment:
I suppose this statement needs to include logic about if the search is visible. Maybe extract logic if search is visible and reused here and around line 246 where the search box is rendered ?